stdlib/runtime/numbers.gr

```diff
@@ -195,7 +178,7 @@ let reducedFraction64 = (x, y) => {
 let safeI32Multiply = (x, y) => {
   let prod = WasmI64.mul(WasmI64.extendI32S(x), WasmI64.extendI32S(y))
   if (WasmI64.gtS(prod, _I32_MAX) || WasmI64.ltS(prod, _I32_MIN)) {
-    throwOverflowError(x)
+    throw Exception.Overflow
```

Comment:
For these Overflows, why doesn't it contain the number that overflowed?

Reply:
Largely it's because I don't think that information is at all useful. 99% of the time if you overflow the value is meaningless (and for what we were doing up until this point, it's definitely meaningless—see https://github.com/grain-lang/grain/issues/301). The other reasons are that it takes a lot to support it—it introduces a dependency on `itoa` for all programs to print that value, and also complicates the base printer quite a bit. I think this is perfectly fine for now, and if we really want we can keep #301 open (but I personally wouldn't bother).